lib/runner/job.ex

```diff
@@ -93,11 +84,12 @@ defmodule ElixirBench.Runner.Job do
     end
   end
 
-  defp get_benchmars_output_path(%Job{id: id}) do
+  def get_benchmars_output_path(%Job{id: id}) do
```

Comment:
should be called `brnchmarks` still with the k :)

Reply:
:heavy_check_mark: changed